Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM bitnami/minideb:jessie
 # Why not Alpine? No glibc. Not willing to shave the yak of compiling with musl right now.
 # Why not busybox? No libpthread, and no package manager to install libpthread.
 
+RUN apt-get update && apt-get upgrade -y 
```

Comment:
I've noticed that we tend to do this a lot and I'm not a huge fan of this though I understand the convenience given our issues with Twistlock. I'm of the mind that a build should be idempotent and a specific git SHA should produce the same artifacts every single time. Anyway, +1 but just wanted to give my $0.02.